crates/bevy_ecs/src/component.rs

```diff
@@ -447,26 +483,33 @@ impl Components {
         index
     }
 
+    /// Returns the number of components registered with this instance.
     #[inline]
     pub fn len(&self) -> usize {
         self.components.len()
     }
 
+    /// Returns `true` if there are no components registered with this instance. Otherwise, this returns `false`.
     #[inline]
     pub fn is_empty(&self) -> bool {
         self.components.len() == 0
     }
 
+    /// Gets the metadata associated with the given component.
+    /// Returns `None` if that component has not been registered with this instance.
```

Comment:
Ah, that's not quite the meaning I intended with that sentence.

If you use a ComponentId from a different world, this will return a garbage result, though it's not undefined behavior unless you call some `unsafe` code with it.